src/dvclive/live.py

```diff
@@ -299,6 +307,33 @@ def log_param(
         """Saves the given parameter value to yaml"""
         self.log_params({name: val})
 
+    def log_artifact(
+        self,
+        path: str,
+    ):
+        from dvc.exceptions import OutputDuplicationError
+
+        """Saves a local file or directory as an artifact"""
+        if not isinstance(path, str):
+            raise InvalidDataTypeError("artifact", type(path))
+
+        if self._dvc_repo is not None:
+            if path in self._outs or self._inside_dvc_exp:
+                stage = self._dvc_repo.commit(path, force=True)
+            else:
+                try:
+                    stage = self._dvc_repo.add(path)
+                except OutputDuplicationError:
+                    raise DuplicateArtifactError(path)
```

Comment:
No strong opinion but I think it might be better to just log a warning and not interrupt the execution.